doc/lightning_tensor/device.rst

```diff
@@ -53,6 +53,8 @@ Check out the :doc:`/lightning_tensor/installation` guide for more information.
 
 .. seealso:: `DefaultTensor <https://docs.pennylane.ai/en/latest/code/api/pennylane.devices.default_tensor.DefaultTensor.html>`__ for a CPU only tensor network simulator device.
 
+Note that it is recommended to create a ``lightning.tensor`` device for each quantum circuit simulation to ensure resources are correctly handled.
+
```

Comment:
```suggestion
Note that as `lightning.tensor` cannot be cleaned up like other state-vector devices because the data is attached to the graph. It is recommended to create a new ``lightning.tensor`` device per circuit to ensure resources are correctly handled.
```